OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -21,7 +21,6 @@ namespace OpenRA.Mods.Common.Traits.Render
 	[Desc("Displays a custom UI overlay relative to the actor's mouseover bounds.")]
 	public class WithDecorationInfo : WithDecorationBaseInfo
 	{
-		[FieldLoader.Require]
 		[Desc("Image used for this decoration. Defaults to the actor's type.")]
 		public readonly string Image = null;
```

Comment:
This would have similar issues then if someone was to not override this value in the future.